setup.py

```diff
@@ -42,7 +42,7 @@
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.4",
-        "qujax >= 0.1",
+        "qujax >= 0.1.1",
```

Comment:
This mean that you will allow all newer version of qujax, are you sure you want to do that? Might be good to do something like `"qujax ~= 0.1.1",`

Reply:
it now requires at least 0.1.2 (I don't think it will work with 0.1.0 or 0.1.1 - on some machines at least)

~= means greater than or equal but still in 0.1?

Reply:
Yes that is the idea